gatsby-browser.js

```diff
@@ -0,0 +1,4 @@
+exports.onClientEntry = () => {
+  // cleanup detected language at least for now when availableLngs evolves
+  typeof window !== 'undefined' && window.localStorage.removeItem('@wappsLng')
```

Comment:
Should this use instead localForage or is there Gatsby specific implementation?

Reply:
This removes item set by gatsby-plugin-i18n, so yes need to use exactly localStorage.